raft/api.go

```diff
@@ -35,5 +35,10 @@ func (s *PublicRaftAPI) Leader() (string, error) {
 	if nil != err {
 		return "", err
 	}
-	return addr.nodeId.String(), nil
+	return addr.NodeId.String(), nil
+}
+
+func (s *PublicRaftAPI) Cluster() []*Address {
+	return append(s.raftService.raftProtocolManager.NodeInfo().PeerAddresses,
+		s.raftService.raftProtocolManager.NodeInfo().Address)
```

Comment:
Can you extract `s.raftService.raftProtocolManager.NodeInfo()` to a local variable, and then reuse that value for accessing both `PeerAddresses` and `Address`? This is because `NodeInfo()` takes a lock and does a bunch of work:

https://github.com/jpmorganchase/quorum/blob/d71daf324590d0033d0fc0ad255cadb702cf2b3b/raft/handler.go#L184-L219